src/app/components/navbar/navbar.component.ts

```diff
@@ -42,6 +42,11 @@ export class NavbarComponent extends SubscriptionsComponent implements OnInit {
 			icon: 'fa-gear',
 			auth: true
 		},
+		{
+			url: '/campus',
+			name: 'Campus Life',
+			icon: 'fa-graduation-cap'
+		},
```

Comment:
Did you mean to move the navbar position over one? This is adding an extra tab without removing the previous one.